README.md

```diff
@@ -239,24 +239,290 @@ One can also do it manually:
 
       $ dotnet new -u Equinox.Templates
 
+<a name="guidance"></a>
 # PATTERNS / GUIDANCE
 
-## Use Strongly typed ids
+<a name="tldr"></a>
+## TL;DR
 
-Wherever possible, the samples strongly type identifiers, particularly ones that might naturally be represented as primitives, i.e. `string` etc.
+1. DO have global strongly typed id types in `namespace Domain`
+2. DONT have global `module Types`. AVOID per Aggregate `module Types` or top level `type` definitions
+3. DO group stuff predictably per `module Aggregate`: `Stream, Events, Reactions, Fold, Decisions, Service, Factory`. And keep grouping within that.
+4. DONT `open <Aggregate>`, `open <Aggregate>.Events`, `open <Aggregate>.Fold`
+5. DO design for idempotency everywhere
+6. DONT use `Result` or a per-Aggregate `type Error`. DO use minimal result types per decision function
+7. DONT pass out `Fold.State` from a `Service`
+8. DONT be a slave to CQRS for all read paths. CONSIDER AllowState. AVOID Query
```

Comment:
Can you clarify `AVOID Query`? I have query modules converting `Fold.State -> Query.Model`

Reply:
```suggestion
8. DONT be a slave to CQRS for all read paths. CONSIDER AllowStale. AVOID Query
```

Reply:
And yes, I had seen that typo!

Reply:
Hopefully covered by https://github.com/jet/dotnet-templates/pull/133/commits/c5c9aa48fe4b1bd9d00dc80022dbc55b1ee7cfcd